x/pki/handler_test.go

```diff
@@ -1942,6 +1955,716 @@ func TestHandler_ApproveX509RootCert_FourApprovalsAreNeeded_FiveTrustees(t *test
 	require.Equal(t, testconstants.RootSerialNumber, approvedCertificate.SerialNumber)
 }
 
```

Comment:
Please add the following positive tests:
- Add:
  - PAA with VID
  - PAI with numeric VID/PID
  - PAI with string VID/PID
  - PAI without PID
- Update
  - PAA with VID
  - PAI with VID/PID
  - PAI without PID  
- Delete
  - PAA
  - PAI